src/main/java/AttendanceTracker/DefaultYear.java

```diff
@@ -0,0 +1,15 @@
+package AttendanceTracker;
```

Comment:
Tia my bad I didn't mean separate classes for each function I meant one class with each of these functions inside.